model/dialogue/hmnet_model.py

```diff
@@ -0,0 +1,199 @@
+from ..base_model import SummModel
+import argparse
+import os
+import torch
+import gzip
+import json
+import uuid
+from nltk import word_tokenize
+import sys
+from ..third_party.HMNet.Models.Trainers.HMNetTrainer import HMNetTrainer
+from ..third_party.HMNet.Utils.Arguments import Arguments
+from ..third_party.HMNet.Models.Networks.MeetingNet_Transformer import MeetingNet_Transformer
+from ..third_party.HMNet.Models.Criteria.MLECriterion import MLECriterion
+
+# TODO: readme -- download the pretraining model,
+#       git clone HMNet before use,
+#       the dependencies that are needed to be installed.
+# TODO: set the role dict, spacy.
```

Comment:
I think for this, you can set a specific commit of another repo as a `module` in our repo. Google `git submodule`